app/src/main/java/com/erwanlbp/calculit/GameConfig.java

```diff
@@ -19,12 +23,25 @@
     public static final String CONFIG_MAX_NUMBER = MainActivity.APPNAME + "CONFIG_MAX_NUMBER";
     public static final int CONFIG_DEFAULT_MAX_NUMBER = 5;
 
-    public GameConfig(int n, int timeToPrint, int maxNumber) {
+    private Difficulty difficulty;
+    public static final String CONFIG_DIFFICULTY = MainActivity.APPNAME + "CONFIG_DIFFICULTY";
+    public static final int CONFIG_DEFAULT_DIFFICULTY = 1000;
+
+    public GameConfig() {
+        this(Difficulty.EASY);
+    }
+
+    public GameConfig(final Difficulty difficulty) {
+        this(CONFIG_DEFAULT_SIZE, difficulty.getValue(), CONFIG_DEFAULT_MAX_NUMBER, difficulty);
+    }
+
+    private GameConfig(final int n, final int timeToPrint, final int maxNumber, Difficulty difficulty) {
```

Comment:
C'est redondant de recevoir timeToPrint et difficulty, c'est la meme chose au final.
Enleve le timeToPrint, et met this.timeToPrint=difficulty.GetTimeToPrint()